roles/metrics-export.lua

```diff
@@ -369,6 +369,10 @@ local function apply_http(conf)
                     httpd:start()
                 else
                     httpd = httpd_role.get_server(target.value)
+                    if httpd == nil then
+                        error(('failed to get server by name %q, check that roles.httpd was' ..
+                              ' already applied'):format(target.value))
```

Comment:
Does this one also covers a case when `roles.httpd` is applied fine, but the server `value` haven't been configured?

Reply:
Or is it validated beforewise in config check step?

Reply:
If there is no server value in `metrics-export` config, it takes default one from httpd.

Reply:
I mean the case when there *is* a server value in `metrics-export`, but no corresponding server in `httpd`.

Reply:
Yeah, it will throw error on validate step:

https://github.com/tarantool/metrics-export-role/blob/8bf5c505b22bfbd2e453f661b8f04cf9adef6101/roles/metrics-export.lua#L158-L167